unordered-containers.cabal

```diff
@@ -41,6 +41,7 @@ tested-with:
 flag debug
   description:  Enable debug support
   default:      False
+  manual:       False
```

Comment:
If the goal is to make the flag manual-only, this should be `True`.

Compare https://cabal.readthedocs.io/en/stable/cabal-package.html#pkg-field-flag-manual.

Reply:
Fixed